packages/orbit-components/src/Breadcrumbs/BreadcrumbsItem/index.tsx

```diff
@@ -68,32 +45,53 @@ const BreadcrumbsItem = ({
   ...props
 }: Props) => {
   return (
-    <StyledBreadcrumbsItem
+    <li
       data-test={dataTest}
       aria-current={active ? "page" : undefined}
       itemProp="itemListElement"
       itemScope
       itemType="http://schema.org/ListItem"
+      className="flex items-center"
     >
-      <StyledBreadcrumbsItemAnchor
-        isClickable={href || onClick}
+      <Component
         href={href}
-        component={component || DefaultComponent}
-        $active={active}
+        component={component}
         onClick={onClick}
         itemScope
         itemType="http://schema.org/WebPage"
         itemProp="item"
-        itemID={id || href}
+        itemID={id ?? href}
         {...props}
+        className={cx(
+          "text-ink-dark",
+          active ? "font-bold" : "font-medium",
+          (onClick != null || href != null) && [
+            active ? "no-underline" : "underline",
+            "cursor-pointer",
+            "border-none",
+            "p-0",
+            "outline-offset-1",
+            "duration-fast transition-colors ease-in-out",
+            "hover:no-underline",
+            "hover:text-product-normal-hover",
+          ],
+        )}
       >
         <span itemProp="name">{children}</span>
-      </StyledBreadcrumbsItemAnchor>
+      </Component>
+
       <meta itemProp="position" content={String(contentKey)} />
-      {!active && (
-        <StyledBreadcrumbsItemIcon ariaHidden reverseOnRtl size="small" color="tertiary" />
+
+      {active !== true && (
```

Comment:
If `active=false` by default, can't this be kept as `!active`? Are we moving away from these kind of assertions for any reason?